### PR TITLE
Fix typos on the Overview page

### PR DESCRIPTION
**Pull request by jadeleafmoon**
Fixes #

- [x] Add documentation.
- [ ] Add tests.
- [ ] Add a changelog entry.
- [x] Add your name in the contributors file.
- [x] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)
- [x] If you added a new configuration setting, update the [`kinto.tpl`](https://github.com/Kinto/kinto/blob/master/kinto/config/kinto.tpl) file with it.

## Features

**Status: Corrections are complete. PR ready for review**

Made corrections such as fixing typos and other grammatical errors found on the [Kinto Overview](https://docs.kinto-storage.org/en/stable/overview.html) page. Getting the first impression right is important!

## Notes

I made a few corrections so far, and found at least ten more changes to make. 

Before continuing, I want to make sure I'm correctly following Kinto's [Contribution Guidelines](https://github.com/Kinto/kinto/blob/master/CONTRIBUTING.md).
When complete, I will make a note in the comments sections of this PR.

### CODE REVIEW

**Comment by jadeleafmoon** [comment]
Hey folks! This is my first contribution to an open-source project and I am excited to help. 
I read the Code of Conduct and [Contributions Guideline](https://github.com/Kinto/kinto/blob/master/CONTRIBUTING.md), and just want to make sure I am doing things correctly.

For example, I opened a PR before it is complete to get some feedback.

Am I good to continue? Any feedback is welcome. 
I have about ten more changes to make to the Overview page.

**Comment by jadeleafmoon** [comment]
@Natim Thank you for the green light. I'll go ahead and continue!

**Comment by jadeleafmoon** [comment]
The PR is ready for review!

Question: 
Do I add an entry in the Changelog? For example:

### 15.1.2 (unreleased)
### Documentation

- Fixed typos on the Overview page.

File: docs/overview.rst [hunk 1]
```diff
@@ -15,23 +15,23 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
 
 We want to help developers focus on the frontend, and we don't want the challenge
 of storing user data to get in their way. The path between a new idea and deploying to
 production should be short!
 
-Also, we believe data belong to the users, and not necessarily to the application authors.
+Also, we believe data belongs to the users, and not necessarily to the application authors.
 Applications should be decoupled from the storage location, and users should be
-able to choose where their personal data are stored.
+able to choose where their personal data is stored.
```
Comment (lesderid) [inline docs/overview.rst:31]: Data can be a plural noun, so this is a matter of style, not correctness.

File: docs/overview.rst [hunk 2]
```diff
@@ -43,14 +43,14 @@ Use cases
 - **A generic Web database**: JSON store for mobile and Web apps, games, or IoT...
 - **Quickly prototype frontend applications**: don't lose time with server stuff,
   the backend for your next single page app is already there.
-- **Applications as static files**: just host your apps on GitHub pages, your storage backend
+- **Applications as static files**: just host your apps on GitHub pages, your backend storage
```
Comment (lesderid) [inline docs/overview.rst:46]: This change looks wrong to me?

File: docs/overview.rst [hunk 3]
```diff
@@ -43,14 +43,14 @@ Use cases
...
 - **Offline-first applications**: data can also be stored locally and published later.
-- **Build collaborative applications** with real time updates and fine-grained permissions.
+- **Build collaborative applications** with real-time updates and fine-grained permissions.
 - **Synchronise application data** between different devices.
-- **Content delivery**: manage remote content or configuration for your apps via an administration UI
-- **Data collection**: easily collect structured data from surveys, forms, analytics.
-- **Store encrypted data** at a location users can control, ensuring better privacy and security.
+- **Content delivery**: manage remote content or configuration for your apps via an administration UI.
+- **Data collection**: easily collect structured data from surveys, forms, and analytics.
+- **Store encrypted data** at a location that users can control, ensuring better privacy and security.
```
Comment (lesderid) [inline docs/overview.rst:53]: This is also a matter of style. Personally I feel like it reads better without the that.

File: docs/overview.rst [hunk 4]
```diff
@@ -246,7 +246,7 @@ For example, when a record is created or updated in a particular collection.
 
 It can send a notification to clients using `WebSockets <https://en.wikipedia.org/wiki/WebSocket>`_
 or fill a queue of messages in `Redis <http://redis.io/>`_ or execute any custom code of your choice,
-like for sending emails or pinging a third-party. For example, at Mozilla, Push notifications are sent to millions of clients using `kinto-megaphone <https://github.com/Kinto/kinto-megaphone>`_.
+like for sending emails or pinging a third party. For example, at Mozilla, push notifications are sent to millions of clients using `kinto-megaphone <https://github.com/Kinto/kinto-megaphone>`_.
```
Comment (lesderid) [inline docs/overview.rst:249]: Also a style thing, "Push" can be capitalised when referring to e.g. Web Push.

**Comment by leplatrem** [comment]
Closing for lack of activity

**Pull request closed without merging**